src/tests/tests_game.go

```diff
@@ -0,0 +1,73 @@
+package tests
+
+import "quarto/game"
+import "testing"
+import "strconv"
+
+// TestDoAMove return correct state
+func TestDoAMove(t *testing.T) {
```

Comment:
Please, be more explicit in your test names like `TestMakeAMoveShouldReturnCorrectState`
This comment is valid for all the tests